setup.cfg

```diff
@@ -30,6 +30,8 @@ install_requires =
     einops
     # fiona 1.5+ required for fiona.transform module
     fiona>=1.5
+    # kornia 0.5.4+ required for kornia.augmentation.AugmentationSequential
+    kornia>=0.5.4
```

Comment:
Is Kornia actually used in torchgeo or just in the tests/tutorials?

Reply:
It's used in the AugmentationSequential wrapper in transforms.py but otherwise only in tests, trainers, and tutorials.